src/SFA.DAS.PR.Application/Permissions/Queries/GetPermissions/GetPermissionsQueryResult.cs

```diff
@@ -4,4 +4,6 @@ namespace SFA.DAS.PR.Application.Permissions.Queries.GetPermissions;
 public class GetPermissionsQueryResult
 {
     public List<Operation> Operations { get; set; } = new();
+    public string ProviderName { get; set; } = null!;
```

Comment:
Tentative: I would just set these to required so there is no need to notify the compiler to ignore any possible null values.

Reply:
ok done